docs/pages/_app.js

```diff
@@ -315,10 +315,7 @@ function AppWrapper(props) {
 
   const activePage = findActivePage(pages, router.pathname);
 
-  let fonts = [
-    'https://fonts.googleapis.com/css?family=Roboto:300,400,400italic,500,700&display=swap',
-    'https://fonts.googleapis.com/css?family=Inter:400,600,700&display=swap',
-  ];
+  let fonts = [];
```

Comment:
This already affects the current landing page. Let's add a TODO instead once we switch.

Reply:
reverted and added todo.